solidity/test/BondedECDSAKeepIntegrationTest.js

```diff
@@ -254,6 +256,9 @@ contract("BondedECDSAKeepFactory", async (accounts) => {
       keepTokenGrant.address
     )
 
+    await TokenStaking.link("GrantStaking", (await GrantStaking.new()).address)
+    await TokenStaking.link("Locks", (await Locks.new()).address)
```

Comment:
Let's keep them next to `MinimumStakeSchedule` linkage.

Reply:
done